example-proxy.ini

```diff
@@ -65,4 +63,18 @@ attribute = sAMAccountName
 # The timeout is specified in seconds.
 # This feature is EXPERIMENTAL.
 enabled = false
-timeout = 3
\ No newline at end of file
+timeout = 3
+
+[preamble-cache]
```

Comment:
I think "search cache" is a bit problematic, since we do not cache *all* LDAP search requests, but only the ones that contain an app marker. "App cache" is not entirely correct either, because we do not cache apps.
... so, naming is hard 😞 But I realize that inventing a lot of new words may overcomplicate everything. So I would probably go for "app cache"?